setup.cfg

```diff
@@ -0,0 +1,45 @@
+[metadata]
+name = labscript_devices
+description = Device drivers for hardware controlled by the labscript suite
+long_description = file: README.md
+long_description_content_type = text/markdown
+author = The labscript suite community
+author_email = labscriptsuite@googlegroups.com
+url = http://labscriptsuite.org
+project_urls = 
+    Source Code=https://github.com/labscript-suite/labscript-devices
+    Download=https://github.com/labscript-suite/labscript-devices/releases
+    Tracker=https://github.com/labscript-suite/labscript-devices/issues
+keywords = experiment control automation
+license = BSD
+classifiers =
+    License :: OSI Approved :: BSD License
+    Programming Language :: Python :: 2.7
```

Comment:
We should consistently remove 2.7 support. At the moment it's been dropped from lyse, runmanager, and runviewer.

Reply:
[definitely](https://www.youtube.com/watch?v=vZw35VUBdzo)